uk.co.electronstudio.ComicReaderUltimate.desktop

```diff
@@ -0,0 +1,9 @@
+[Desktop Entry]
+Version=1.0
+Type=Application
+Terminal=false
+Exec=/app/bin/cru.sh %f
```

Comment:
Exec shouldn't contain a full path.

Reply:
If I remove the path and attempt to launch it says:

   `bwrap: execvp cru.sh: No such file or directory`

Reply:
I also tried following the instructions at https://docs.flatpak.org/en/latest/conventions.html which says to put
`Exec=uk.co.electronstudio.ComicReaderUltimate` but that doesn't work either.  It says `WARNING: Binary not found for Exec line` during the build.

Reply:
this imply the executable is `uk.co.electronstudio.ComicReaderUltimate` but you have a wrapper script. (it's fine to have one). Also it's just an example.

Reply:
ok i figured out i can use  "--env=PATH=/app/bin" to get it to work without full path in exec

Reply:
this should be set by default. weird.

Reply:
ah i think it probably was set by default but i overrode it because i copied an example from org.freedesktop.Sdk.Extension.openjdk